varinfo/var_info.py

```diff
@@ -522,48 +497,26 @@ def _read_dataset(self, file_path: str):
         self.dataset = ET.fromstring(dmr_content)
         self.namespace = get_xml_namespace(self.dataset)
 
-    def _set_global_attributes(self):
-        """Extract all global attributes from a `.dmr` file. First this method
-        searches for a root level Attribute element with name
-        "HDF5_GLOBAL". If this is present, it is assumed to be a container
-        for the global attributes. If "HDF5_GLOBAL" is absent, the global
-        attributes are assumed to be direct children of the root Dataset
-        element in the XML tree. All child Attribute elements children with
-        a type property corresponding to a DAP4 variable type are placed in
-        an output dictionary. If the type is not recognised by the DAP4
-        protocol, the attribute is assumed to be a string.
+    def _set_short_name(self):
+        """Iterate through all suggested locations for the collection short
+        name, as listed in the configuration file. For each location, perform a
+        search for an XML element in the DMR document for that element and, if
+        found, retrieve the value of that element.
 
         """
 
-        def save_attribute(output, group_path, attribute):
```

Comment:
Unhelpful diff here. The removed code is the content of `_set_global_attributes`, which is now removed. It's sandwiched between the new `_set_short_name` method.